schemas/manifest.fbs

```diff
@@ -74,6 +88,17 @@ table ManifestV1 {
     wal_object_store_uri: string;
 }
 
+table CompactedSstId {
+    high: uint64;
+    low: uint64;
+}
+
+table CompactedSsTable {
```

Comment:
Would this cause any issue for SSTs created in older SlateDB versions?

Reply:
Yep, FB optional field additions are safe _as long as they're at the end of the schema_.

https://flatbuffers.dev/evolution/#addition

This change should be safe.